[IMP] attendances: fixed typo and added details section (#8068)
odoo/documentation: content/applications/hr/attendances.rst

The default view presents the current day's information. To present the information for the
:guilabel:`Week`, :guilabel:`Month`, or :guilabel:`Year`, click on the :guilabel:`Day` button to
reveal a drop-down, displaying those other options. Select the desired view, and the dashboard
updates, presenting the selected information. To change the :guilabel:`Day`, :guilabel:`Week`,

> :guilabel:`Month`, or :guilabel:`Year` presented, click the left or right :guilabel:`arrow` buttons

Reviewer: ```suggestion
:guilabel:`Month`, or :guilabel:`Year` presented, click the left or right :guilabel:`arrow` buttons
```
Would arrow emojis work here?

Author: I changed it to use an icon instead of an emoji, because the emojis are in a bright blue box, which I don't personally like since it's a different look from our UI.

on either side of the drop-down button. To jump back to a view containing the current day, click the

> :guilabel:`Today` button. The refreshed the dashboard, presenting information containing the current

Reviewer: ```suggestion
:guilabel:`Today` button. This refreshes the dashboard, presenting information containing the current
```

.. _attendances/filters-groups:

Filters and groups
------------------

> To filter the results in the overview dashboard or to present different groups of information, click

Reviewer: ```suggestion
To filter the results in the overview dashboard, or to present different groups of information, click
```

> the drop-down button in the :guilabel:`Search` bar above the dashboard, then select one of the

Reviewer: ```suggestion
the drop-down button in the :guilabel:`Search` bar above the dashboard, and select one of the
```

.. _attendances/errors:

Attendance log details
----------------------

Odoo captures various time and location details when a user checks in and out. The specific

> details provided are deterined by the method the user checked in and out.

Reviewer: ```suggestion
details provided are determined by the method the user checked in and out.
```

To view the specific check in and/or check out details for an employee, click on an individual entry
in the overview dashboard.

A detailed attendance log for the user appears in a pop-up window. To close the detailed attendance
log, click the :guilabel:`Save & Close` button in the bottom-left corner of the form.

The detailed attendance log contains the following information:

Main details
~~~~~~~~~~~~

- :guilabel:`Employee`: the name of the employee.
- :guilabel:`Check In`: the date and time the employee checked in.
- :guilabel:`Check Out`: the date and time the employee checked out. This only appears if the
  employee has checked out.
- :guilabel:`Worked Hours`: the total amount of time the employee logged for the day, in an hour and
  minute format (XX:XX). This value calculates all the checks in and check outs for the day, if the
  employee checked in and out multiple times.
- :guilabel:`Extra Hours`: any extra hours the employee logged that is beyond their expected working
  hours.

Check in/check out details
~~~~~~~~~~~~~~~~~~~~~~~~~~

The following information appears for both the :guilabel:`Check In` and :guilabel:`Check Out`
sections.

> - :guilabel:`Mode`: the method the attendance information was gathered. :guilabel:`Systray` is

Reviewer: ```suggestion
- :guilabel:`Mode`: the method with which the attendance information was gathered. :guilabel:`Systray` is
```

.. _attendances/errors:

Attendance log details
----------------------

Odoo captures various time and location details when a user checks in and out. The specific details
provided are determined by the method the user checked in and out.

To view the specific check in and/or check out details for an employee, click on an individual entry
in the overview dashboard.

A detailed attendance log for the user appears in a pop-up window. To close the detailed attendance
log, click the :guilabel:`Save & Close` button in the bottom-left corner of the form.

The detailed attendance log contains the following information:

Main details
~~~~~~~~~~~~

- :guilabel:`Employee`: the name of the employee.
- :guilabel:`Check In`: the date and time the employee checked in.
- :guilabel:`Check Out`: the date and time the employee checked out. This only appears if the
  employee has checked out.
- :guilabel:`Worked Hours`: the total amount of time the employee logged for the day, in an hour and

>   minute format (XX:XX). This value calculates all the checks in and check outs for the day, if the

Reviewer 2: ```suggestion
  minute format (HH:MM). This value calculates all the checks in and check outs for the day, if the
```
